app/controllers/api/custom_buttons_controller.rb

```diff
@@ -14,10 +14,15 @@ def create_resource(_type, _id, data)
     end
 
     def edit_resource(type, id, data)
+      return if data.empty?
       custom_button = fetch_custom_button(type, id)
       updated_data = data['resource'] || data
-      updated_data['resource_action'] = find_or_create_resource_action(updated_data['resource_action']) if updated_data.key?('resource_action')
-      custom_button.update!(updated_data.deep_symbolize_keys) if data.present?
+      resource_action = updated_data['resource_action']
+      create_or_update_resource_action(updated_data['resource_action']) if resource_action.present?
+      updated_data.except!('resource_action')
+      custom_button.update!(updated_data.deep_symbolize_keys) if updated_data.present?
```

Comment:
@lpichler This line will change `ae_attributes` in `custom_button.resource_action` from :
```ruby
#<ResourceAction:0x00007fc0dff5fc10
 id: 7603,
 action: nil,
 dialog_id: nil,
 resource_id: 70,
 resource_type: "CustomButton",
 created_at: Wed, 19 Feb 2020 08:45:52 UTC +00:00,
 updated_at: Wed, 19 Feb 2020 13:36:23 UTC +00:00,
 ae_namespace: "SYSTEM",
 ae_class: "PROCESS",
 ae_instance: "Request",
 ae_message: "create",
 ae_attributes: {:pepek=>"VALUE", :KAREL=>"VALUES", :request=>"PEPEK", :service_template=>nil},
 configuration_template_id: nil,
 configuration_template_type: nil>
```
to this:
```ruby
#<ResourceAction:0x00007fc0dff5fc10
 id: 7603,
 action: nil,
 dialog_id: nil,
 resource_id: 70,
 resource_type: "CustomButton",
 created_at: Wed, 19 Feb 2020 08:45:52 UTC +00:00,
 updated_at: Wed, 19 Feb 2020 13:36:52 UTC +00:00,
 ae_namespace: "SYSTEM",
 ae_class: "PROCESS",
 ae_instance: "Request",
 ae_message: "create",
 ae_attributes: {:request=>"PEPEK", :service_template=>nil, :hosts=>nil},
 configuration_template_id: nil,
 configuration_template_type: nil>
```
`custom_button.save` works as expected. Any ideas what's wrong?